### PR TITLE
Increment NetCurrent and NetPrevious

### PR DESCRIPTION
**Pull request by mthalman**

### CODE REVIEW

**Comment by mthalman** [comment]
I'm unsure of the rollout of this. It would force repos to take a dependency on a newer SDK in order to get the arcade dependency.

**Comment by mmitche** [comment]
> I'm unsure of the rollout of this. It would force repos to take a dependency on a newer SDK in order to get the arcade dependency.

I think we should wait on this a bit until the net9 SDK is stable. Then I would do the following:
1. Update the SDK in arcade to net9
2. Flow to repos
3. Update Properties
4. Flow to repos

Separating the new SDK usage from the tfm update should reduce some noise.

**Comment by ViktorHofer** [comment]
Agreed. I think it's weeks too early to increment these TFM properties across the stack. Probably a month or so to let the stack first stabilize (i.e. dotnet/sdk and installer).

**Review by ViktorHofer — COMMENTED** [review]
Please change L40 to:

```
<NetToolMinimum>net8.0</NetToolMinimum>
<NetToolMinimum Condition="'$(NetToolMinimum)' == '$(NetToolCurrent)'" />
```

File: eng/common/tools.ps1 [hunk 1]
```diff
@@ -601,7 +601,7 @@ function InitializeBuildTool() {
       ExitWithExitCode 1
     }
     $dotnetPath = Join-Path $dotnetRoot (GetExecutableFileName 'dotnet')
-    $buildTool = @{ Path = $dotnetPath; Command = 'msbuild'; Tool = 'dotnet'; Framework = 'net8.0' }
+    $buildTool = @{ Path = $dotnetPath; Command = 'msbuild'; Tool = 'dotnet'; Framework = 'net9.0' }
```
Comment (ViktorHofer) [inline eng/common/tools.ps1:604]: I think this change needs to wait until the updated Arcade.Sdk is consumed in Arcade.

**Comment by ViktorHofer** [comment]
I pushed to your branch to reset NetPrevious. Hope that's OK.

**Comment by mthalman** [comment]
This is replaced by https://github.com/dotnet/arcade/pull/14279.

**Pull request closed without merging**